qe_metrics/utils/issue_utils.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import annotations
+from datetime import date, datetime
+
+from jira import Issue
+
+from simple_logger.logger import get_logger
+from typing import TYPE_CHECKING, List
+
+if TYPE_CHECKING:
+    from qe_metrics.libs.database import Database
+
+LOGGER = get_logger(name=__name__)
+
+
+def check_customer_escaped(issue: Issue) -> bool:
+    """
+    Check if the issue is customer escaped.
+
+    Args:
+        issue (Issue): Jira Issue
+
+    Returns:
+        bool: True if the issue is customer escaped, False otherwise.
+    """
+    try:
+        return float(issue.fields.customfield_12313440) > 0
```

Comment:
Can you set a mapping of name: field and work with it?
`customfield_12313440` means nothing.